src/main/java/com/junwoo/ott/domain/video/repository/VideoJpaRepository.java

```diff
@@ -1,8 +1,16 @@
 package com.junwoo.ott.domain.video.repository;
 
 import com.junwoo.ott.domain.video.entity.Video;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
 
 public interface VideoJpaRepository extends JpaRepository<Video, Long> {
 
+  @Query("select v from Video v where v.deletedAt is null order by v.createdAt desc")
+  Page<Video> getVideo(Pageable pageable);
```

Comment:
메소드 이름에 s가 붙으면 좋을 것 같습니다.
```suggestion
  Page<Video> getVideos(Pageable pageable);
```

Reply:
넵!